types/react/index.d.ts

```diff
@@ -1284,7 +1284,7 @@ declare namespace React {
     // Event Handler Types
     // ----------------------------------------------------------------------
 
-    type EventHandler<E extends SyntheticEvent<any>> = { bivarianceHack(event: E): void }["bivarianceHack"];
+    type EventHandler<E extends SyntheticEvent<any>> = { bivarianceHack(this : void, event: E): void }["bivarianceHack"];
```

Comment:
<https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45834/checks#step:5:1917>

```suggestion
    type EventHandler<E extends SyntheticEvent<any>> = { bivarianceHack(this: void, event: E): void }["bivarianceHack"];
```